src/components/carbon-provider/carbon-provider.d.ts

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+
+import { ThemeObject } from "../../style/themes/base";
+
+export interface CarbonProvider {
+  theme?: ThemeObject;
+  children: React.ReactNode;
+}
```

Comment:
The interface alone isn't enough here, we'll need to add something like:

```jsx
declare function CarbonProvider(props: CarbonProviderProps): JSX.Element;
export default CarbonProvider;
```

Reply:
you are right, changed, thanks